packages/amplify-provider-awscloudformation/src/initializer.ts

```diff
@@ -53,6 +55,37 @@ export async function run(context) {
     const authRoleName = `${stackName}-authRole`;
     const unauthRoleName = `${stackName}-unauthRole`;
 
+    const configuration = {
+      authRole: {
+        roleName: authRoleName,
+      },
+      unauthRole: {
+        roleName: unauthRoleName,
+      },
+    };
+
+    const noOverrideMsg = '';
+    try {
+      const backendDir = pathManager.getBackendDirPath();
+      const overrideFilePath = path.join(backendDir, 'awscloudformation', 'build', 'override.js');
+      const overrideCode: string = await fs.readFile(overrideFilePath, 'utf-8');
+      if (overrideCode) {
+        const sandboxNode = new vm.NodeVM({
+          console: 'inherit',
+          timeout: 5000,
+          sandbox: {},
+          require: {
+            context: 'sandbox',
+            builtin: ['path'],
+            external: true,
+          },
+        });
+        sandboxNode.run(overrideCode).override(configuration);
+      }
+    } catch (e) {
+      printer.debug(`Unable to apply auth role overrides: ${e.message}`);
```

Comment:
`.debug` hides the error message behind the `--debug` flag. Change this to `.error`, or consider throwing if it makes more sense to fast fail

Reply:
I think —debug is correct. In the common case we expect to log this debug line.

Reply:
That behavior was marked as a bug: https://github.com/aws-amplify/amplify-cli/issues/9695